mono/mini/interp/interp.h

```diff
@@ -6,6 +6,13 @@
 #define __MONO_MINI_INTERPRETER_H__
 #include <mono/mini/mini.h>
 
+typedef struct _MonoInterpStackIter MonoInterpStackIter;
+
+/* Needed for stack allocation */
+struct _MonoInterpStackIter {
+	gpointer dummy [8];
+};
```

Comment:
that's a bit confusing, why not putting `StackIter` from `interp.c` here instead?